tests/AppBundle/Controller/MailingListControllerTest.php

```diff
@@ -76,7 +78,7 @@ private function generateListCountChars(Client $client, string $type)
     {
         $crawler = $this->goTo('/kontrollpanel/epostlister', $client);
         $form = $crawler->selectButton('Generer')->form();
-        $form['generate_mailing_list[semester]'] = 2;
+        $form['generate_mailing_list[semester]'] = 1;
```

Comment:
Here is the other test that had its id changed. I suspect that it is supposed to reference the current semester, which in an earlier state had id = 2. Currently, semester with id = 2 references the previous semester, and that semester does not have any entries in the Assistant mailing list (whereas the current semester has 2)

Reply:
⚠️ THIS SHOULD BE APPROVED BEFORE MERGE ⚠️

Reply:
The mentioned change was also introduced with `Fjern department fra semester`, #1048 by @sigtot :

Before. note, here the current semester will have id = 2, as it is the second entity to be persisted to the db:
![Screenshot 2020-09-24 at 12 23 59](https://user-images.githubusercontent.com/31702423/94133404-e32e4f80-fe60-11ea-97fb-22cc568533bb.png)

After. Here, the current semester is persisted first and will have id = 1:
![Screenshot 2020-09-24 at 12 24 12](https://user-images.githubusercontent.com/31702423/94133510-078a2c00-fe61-11ea-82ea-e0b00ce44d20.png)